odd_collector/domain/plugin.py

```diff
@@ -296,6 +296,11 @@ class CouchbasePlugin(BasePlugin):
     num_sample_values: Optional[int] = 10
 
 
+class SQLitePlugin(BasePlugin):
+    data_source: str
```

Comment:
Use path type from pydantic for validations https://docs.pydantic.dev/latest/usage/types/#standard-library-types

Reply:
Added appropriate type.